barman/cloud_providers/aws_s3.py

```diff
@@ -102,6 +103,7 @@ def __init__(
         self,
         url,
         encryption=None,
+        aws_irsa=False,
```

Comment:
please relocate this new argument at the end of the argument list to preserve the original signature @smcaine